docs/pages/admin-guides/management/export-audit-events.mdx

```diff
@@ -28,3 +28,6 @@ events to your solution of choice:
 - [Monitor Teleport Audit Events with Splunk](./export-audit-events/splunk.mdx):
   How to configure the Event Handler plugin to send logs to Splunk's Universal
   Forwarder so you can explore your audit events in Splunk.
+- [Monitor Teleport Audit Events with Datadog](./export-audit-events/datadog.mdx):
+  How to configure the Event Handler plugin to send logs to Datadog's Agent via Fluentd
+  so you can explore your audit events in Datadog.
```

Comment:
```suggestion
  How to configure the Event Handler plugin to export audit logs to Datadog via Fluentd.
```
Since the Datadog Agent isn't involved in the setup shown in the export guide (it's event-handler --> Fluentd --> Datadog ingestion endpoint).